src/utils/fetchData.js

```diff
@@ -1,19 +1,21 @@
 import queryPerfData from '@mozilla-frontend-infra/perf-goggles';
-import { BENCHMARKS, CONFIG } from '../config';
+import linux64 from '../configuration/js-team/linux64';
 import prepareData from './prepareData';
 import { convertToSeconds } from './timeRangeUtils';
+import DASHBOARD_CONFIG from '../configuration/appDefaults';
+import platforms from '../configuration/js-team/index';
 
 const fetchData = async (platform, benchmark, range) => {
```

Comment:
This possibly the piece of code that has caused the most trouble to you. I did not write it well.
I coupled in this function two distinct matters: configuration manipulation and fetching data.

Let me create a branch with some code that will change this logic and make it more sane.
Tackle the other issues. I should have some code for you later today.